### PR TITLE
Update 2021-03-08-week-1.md - Small edits

### PR DESCRIPTION
**Pull request by vsmart**

### CODE REVIEW

File: content/blog/2021-03-08-week-1.md [hunk 1]
```diff
@@ -3,27 +3,27 @@ title: 'Week 1 in review'
 date: 2021-03-08
 ---
```
Comment (vsmart) [inline content/blog/2021-03-08-week-1.md:5]: Add an introductory sentence or two to let the reader know what's going on. Something like:
```suggestion
After months of preparation, this week the Feminizmap project kicked off! The first week ... 
```

File: content/blog/2021-03-08-week-1.md [hunk 2]
```diff
@@ -3,27 +3,27 @@ title: 'Week 1 in review'
...
 
-After the initial kickoff workshop organized by Prototype Fund past Monday and the last bits of paperwork out of the way, the first week was all about setting up stuff.
+After the initial kickoff workshop organized by Prototype Fund past Monday and the last bits of paperwork out of the way, the first week was all about setting up the initial infrastructure.
 
 ## What did I do?
```
Comment (vsmart) [inline content/blog/2021-03-08-week-1.md:9]:
```suggestion
This first week, I focused on setting up the initial groundwork for testing, deployment, and the basic app skeleton.
```

File: content/blog/2021-03-08-week-1.md [hunk 3]
```diff
@@ -3,27 +3,27 @@ title: 'Week 1 in review'
...
+While setting up automation of things I decided to setup up continuous delivery (CD) to a staging environment on a hobby dyno on Heroku as well. I want the production app to be deployed on a VPS via docker (-compose) but it will take some time until that is all done and available. So, why not have an always up-to-date version running for easy testing and have an alternative way to deploying? Exactly!
 
 For documentation and a developer blog I set up this page, using [Hugo](https://gohugo.io/) as static site generator and the [Hugo Whisper Theme](https://github.com/zerostaticthemes/hugo-whisper-theme) and made some adjustments. Github Actions also helps here with automatic deployment to GiHub Pages.
 
-Too much GitHub? Probably, but it definitely helps getting setup quickly. I'm more familiar with the interface than with GitLab's plus I don't want the hassle to maintain (and pay for) my own infrastructure at this point.
+Too much GitHub? Probably, but it definitely helps getting setup quickly. I'm more familiar with the interface than with GitLab's, plus I don't want the hassle to maintain (and pay for) my own infrastructure at this point.
```
Comment (vsmart) [inline content/blog/2021-03-08-week-1.md:26]: I think most readers won't know what you mean by "too much Github". Either remove or clarify this.

**Comment by lislis** [comment]
Thanks for the pointers!

**Pull request closed without merging**